phosphord/README.md

```diff
@@ -0,0 +1,28 @@
+# PhosphorD
+
+PhosphorD is a local forwarder, like StatsD, which receives traces from the Phosphor client, and forwards to the [Phosphor server](https://github.com/mondough/phosphor/tree/master/phosphor).
+
+Currently this receives Traces over UDP, which prevents clients blocking, but is reasonably reliable on a local machine. In the event this blocks, traces will be dropped and lost.
+
+A future improvement would make this configurable to read from local files, mirroring the behaviour of Dapper Daemons as described in the [Google Dapper](https://research.google.com/pubs/pub36356.html) paper.
```

Comment:
Yeah that would be nice. We certainly don't need to target windows for the time being, and the `dockerize` branch has container support if needed.